x-pack/plugins/observability/server/lib/rules/custom_threshold/register_custom_threshold_rule_type.ts

```diff
@@ -110,6 +110,7 @@ export function thresholdRuleType(
     name: i18n.translate('xpack.observability.threshold.ruleName', {
       defaultMessage: 'Custom threshold (Beta)',
     }),
+    fieldsForAAD: CUSTOM_THRESHOLD_AAD_FIELDS,
```

Comment:
do we need to merge this with ?

`import { O11Y_AAD_FIELDS } from '@kbn/infra-plugin/common/constants';`

Reply:
I thought about it but since we will deprecate the infra rules in the future, I thought having these fields separately makes more sense.

Reply:
let's keep it for now. if it makes sense we can move in future